home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/InvitationStatusAttributeConverter.java

```diff
@@ -18,6 +18,8 @@ public String convertToDatabaseColumn(InvitationStatus attribute) {
                 return "accepted";
             case DECLINED:
                 return "declined";
+            case DEACTIVATED:
```

Comment:
Can we do here the same as we've done for RoleEnum?

Reply:
Okay